update_docker_image.sh

```diff
@@ -0,0 +1,10 @@
+set -e
+
+# DEVELOPER: update this field anytime you make a new docker image
+VERSION="2.0"
```

Comment:
would it make sense to force the developer to input the version number? I worry about folks not realizing they need to update it. 

in other places I put this version in the code itself and have this script pull it out - admittedly also not a great solution but more likely for the developer to see it than here, in my opinion.

Reply:
oh I see this is for the top-level Dockerfile. I thought we were going to remove this and have a docker image per script. but this works, too. 

in this case, since all scripts are included in this image, I agree it makes sense to have this here, but I'd want to do something more to force the developer to increment the version. 

the nicest thing would be a prompt that checks for existing builds with that tag and requires the developer/user to confirm that they want to overwrite the existing build with that tag (which will likely happen during development, so we don't want to auto-increment). what do you think?

Reply:
Yeah - I saw that you'd done that for twistcap, but as you said, it was top-level, so there wasn't specific code to pull from. I could instead just tag this as workspace_cleanup and move the script there. I think I got sidetracked because there wasn't a script for the top-level Docker.
I think a prompt is good. That is what my previous team did for this sort of thing. 
AND/OR should I make a workspace_cleanup-specific image? (for now, it'll be just a copy of the top level, but may diverge one day - and then I wouldn't copy in the whole scripts dir, which felt like overkill...)

Reply:
I'd say do the minimum necessary for this PR - so don't split it out now if there's no need. Can always be done in future.